sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c

```diff
@@ -373,14 +360,13 @@ static int on_received(void* context, char* topicName, int topicLen, MQTTClient_
 
   // Parse the incoming message topic and check to make sure it is a twin message
   az_iot_hub_client_twin_response twin_response;
-  if (az_failed(az_iot_hub_client_twin_parse_received_topic(&client, topic_span, &twin_response)))
+  if (az_failed(
+          rc = az_iot_hub_client_twin_parse_received_topic(&client, topic_span, &twin_response)))
   {
-    printf("Topic is not a twin message.\n");
+    printf("Topic is not a twin message, az_result return code %04x\n", rc);
```

Comment:
`%04x`
Is this just for readability's sake?

Reply:
Yes.  
So for the error AZ_ERROR_ITEM_NOT_FOUND, the print out is 8001004 (which follows the logic in az_result.h) instead of -2147418108 which is cryptic at face value.